dev-tools/packaging/packages.yml

```diff
@@ -892,20 +915,23 @@ specs:
             mode: 0755
 
     - os: linux
+      arch: amd64
       types: [docker]
       spec:
         <<: *agent_docker_spec
+        <<: *docker_ubi_spec
         <<: *elastic_docker_spec
         <<: *elastic_license_for_binaries
         files:
           '{{.BeatName}}{{.BinaryExt}}':
             source: ./build/golang-crossbuild/{{.BeatName}}-{{.GOOS}}-{{.Platform.Arch}}{{.BinaryExt}}
 
     - os: linux
+      arch: arm64
       types: [docker]
       spec:
-        <<: *agent_docker_spec
-        <<: *docker_ubi_spec
+        <<: *agent_docker_arm_spec
+        <<: *docker_arm_ubi_spec
```

Comment:
Is `jq` for ARM in the ubi image needed at the end? we are installing the "linux64" version now.

Reply:
Now we are installing the appropriate version.